contracts/token/ERC721/extensions/ERC721Consecutive.sol

```diff
@@ -56,7 +56,11 @@ abstract contract ERC721Consecutive is IERC2309, ERC721 {
         address owner = super._ownerOf(tokenId);
 
         // If token is owned by the core, or beyond consecutive range, return base value
-        if (owner != address(0) || tokenId > type(uint96).max) {
+        if (
+            owner != address(0) ||
+            tokenId > (uint256(type(uint96).max) + _firstConsecutiveId()) ||
```

Comment:
this feels wrong. The lookup checkpoint (`_sequentialOwnership`)  only support id that are uint96. So if a tokenId is greater than that, we should not try to do the lookup.

said otherwize, `_firstConsecutiveId` should be reducing the range of tokens that can be sequentially minted, not moving it.

Reply:
in that case we should not need `+ _firstConsecutiveId()` since it overflows, I'll remove that and test